packages/test-utils/src/server.ts

```diff
@@ -56,7 +56,7 @@ export async function startServer(): Promise<void> {
       METRICS_PORT: process.env.METRICS_PORT || "3003",
       METRICS_HOST: "127.0.0.1", // Secure binding for tests
       TEST_MODE: "true",
-      LOG_LEVEL: "debug",
+      LOG_LEVEL: process.env.LOG_LEVEL || "debug",
```

Comment:
this is the root of the fix. we werent respecting the LOG_LEVEL in the `.env.test` file.